cpu/arm7_common/periph/pm.c

```diff
@@ -8,6 +8,7 @@
 
 /**
  * @ingroup     cpu_arm7_common
+ * @ingroup     drivers_periph_pm
```

Comment:
A file can't be in two groups in doxygen. I know this isn't significant for this PR, since C-files are not compiled by doxygen, but then again, it is because then we could leave the `@ingroup` tags as is (or at least unify them, so that they either all belong to the CPU's group [which I would prefer] or the periph's group)